samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -173,16 +173,17 @@ def _prompt_run_bootstrap_within_pipeline_init(self, stage_names: List[str], req
                 return True
         else:
             click.echo(
-                Colored().yellow(
-                    dedent(
-                        """\
-                        If you want to setup stages before proceed, please quit the process using Ctrl+C.
-                        Then you can either run 'sam pipeline bootstrap' to setup a stage
-                        or re-run this command with option '--bootstrap' to enable stage setup.
-                        """
-                    )
+                dedent(
+                    """\
+                    To set up stage(s), please quit the process using Ctrl+C and use one of the following commands:
+                    sam pipeline init --bootstrap		To be guided through the stage and config file creation process.
+                    sam pipeline bootstrap				To specify details for an individual stage.
```

Comment:
why they are not aligned
<img width="871" alt="image" src="https://user-images.githubusercontent.com/3804518/125709289-2f77f06d-51ba-4f95-8182-12ab7721c81b.png">